app/service/search_index.rb

```diff
@@ -0,0 +1,17 @@
+class SearchIndex
+  def self.add(record)
+    if search_index_callbacks_enabled?
```

Comment:
I added a logger step with a searchable tag, `[ELASTICSEARCH]` so we can find it easily.